### PR TITLE
Added `CONTRIBUTING.md` and `BUILDING.md` files

### LINKED ISSUE #144: Translation plan
Write a plan or sequence of actions for translation into new languages. For example, in discussions.

### PR DESCRIPTION
**Pull request by IraSoro**
Closed #144 

I added a `CONTRIBUTING.md` file in which I described the steps for translation into new languages.
I added the `BUILDING.md` file. I moved the build information from the `README.md` file into it.
And edited the `README.md` file.

### CODE REVIEW

**Review by imblowfish — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Contributing
+
+## Translations
+
+We welcome contributions for translating our application into various languages. If you'd like to help with translations, please follow these steps.
+
+### Steps
+
+1. Look for the `/utils/translations` directory to find existing translations. If you don't find the language you wish to translate the application into, proceed to step 2.
```
Comment (imblowfish) [inline CONTRIBUTING.md:9]: Maybe `if you didn't find the language you interested in then proceed to step 2`

Comment (imblowfish) [inline CONTRIBUTING.md:9]: `/utils/translations` looks like an absolute unix path. Maybe `./src/utils/translations`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Translations
+
+We welcome contributions for translating our application into various languages. If you'd like to help with translations, please follow these steps.
+
+### Steps
+
+1. Look for the `/utils/translations` directory to find existing translations. If you don't find the language you wish to translate the application into, proceed to step 2.
+
+2. Create a new file for the language by duplicating the `ru.ts` file in the `/translations` directory. Name it using the language code (e.g., `fr.ts` for French translations).
```
Comment (imblowfish) [inline CONTRIBUTING.md:11]: `./src/utils/translations/ru.ts`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+3. Translate the content within the file into your desired language.
```
Comment (imblowfish) [inline CONTRIBUTING.md:13]: Translate the keys into your language

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+4. In the directory `/utils` open the file `translation.ts`. Import the generated file like this:
```
Comment (imblowfish) [inline CONTRIBUTING.md:15]: Import translation file in  the `./src/utils/translations.ts`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+   ```ts
+   import en from "./translations/en";
+   import ru from "./translations/ru";
+   ```
+
+5. Add the language to the `supportedLanguages` variable like this:
```
Comment (imblowfish) [inline CONTRIBUTING.md:22]: Add the language to the `supportedLanguages` map like this, to show this language in the app's menu

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+       en: {
+         translation: en,
+       },
+       ru: {
+         translation: ru,
+       },
+   },
+   ```
+
+7. In the directory `/utils` open the file `datetime.ts`. Add the locale to the `locales` variable like this:
```
Comment (imblowfish) [inline CONTRIBUTING.md:44]: `./src/utils/datetime.ts`

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+   ```ts
+   const locales = new Map([
+     ["en", enUS],
+     ["ru", ru],
+   ]);
+   ```
+
+8. Submit a Pull Request with the translated file.
```
Comment (imblowfish) [inline CONTRIBUTING.md:53]: I'm not sure, but maybe `Make a Pull Requests with the translated file`?

File: README.md [hunk 8]
```diff
@@ -50,77 +50,8 @@ See the [open issues](https://github.com/IraSoro/peri/issues) and [project](http
 
 ### How to build
 
-Install dependencies:
+You can find full instruction about the project building in [BUILDING.md](https://github.com/IraSoro/peri/blob/master/BUILDING.md) file.
```
Comment (imblowfish) [inline README.md:53]: `You can find the full instruction about the project building in`

**Review by imblowfish — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (IraSoro) [reply CONTRIBUTING.md:9]: Ок

Comment (IraSoro) [reply CONTRIBUTING.md:9]: Ok

File: CONTRIBUTING.md (hunk 2 above)
Comment (IraSoro) [reply CONTRIBUTING.md:11]: Fixed

File: CONTRIBUTING.md (hunk 3 above)
Comment (IraSoro) [reply CONTRIBUTING.md:13]: Fixed

File: CONTRIBUTING.md (hunk 4 above)
Comment (IraSoro) [reply CONTRIBUTING.md:15]: Fixed

File: CONTRIBUTING.md (hunk 5 above)
Comment (IraSoro) [reply CONTRIBUTING.md:22]: Fixed

File: CONTRIBUTING.md (hunk 6 above)
Comment (IraSoro) [reply CONTRIBUTING.md:44]: Fixed

File: README.md (hunk 8 above)
Comment (IraSoro) [reply README.md:53]: Fixed

**Pull request merged**